.github/ISSUE_TEMPLATE_tmpl/1_broken_site.yml

```diff
@@ -12,7 +12,7 @@ body:
       options:
         - label: I'm reporting that yt-dlp is broken on a **supported** site
           required: true
-        - label: I've verified that I'm running yt-dlp version **%(version)s** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
+        - label: I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
```

Comment:
How about:
``
I've updated yt-dlp to [nightly or master](https://github.com/yt-dlp/yt-dlp#update-channels) using `yt-dlp --update-to nightly` or `yt-dlp --update-to master`
``

Reply:
`--update-to` may not be an option for the user and should not be listed as a requirement

Reply:
How about ``Update yt-dlp to nightly or master. Instructions to do that can be found [here](https://github.com/yt-dlp/yt-dlp#update-channels), although the easiest way is to just run `yt-dlp --update-to nightly` ``
Preview:
- [ ] Update yt-dlp to nightly or master. Instructions to do that can be found [here](https://github.com/yt-dlp/yt-dlp#update-channels), although the easiest way is to just run `yt-dlp --update-to nightly`

Reply:
Same with what bashonly said - it would be confusing. We are already pointing to the instructions, I think what is present is fine.